_format/1.0/index.md

```diff
@@ -949,10 +950,11 @@ GET /articles/1?include=comments.author HTTP/1.1
 Accept: application/vnd.api+json
 ```
 
-> Note: Because [compound documents][compound document] require full linkage,
-intermediate resources in a multi-part path must be returned along with the leaf
-nodes. For example, a response to a request for `comments.author` should
-include `comments` as well as the `author` of each of those `comments`.
+> Note: Because [compound documents][compound document] require full linkage
+(except when sparse fieldsets are in use), intermediate resources in a multi-part
```

Comment:
Sparse fieldsets don't just need to be in use, they also need to exclude the linkage fields. Perhaps this is more clear:

> (except when linkage fields are excluded from sparse fieldsets)